packages/main/src/themes/Button.css

```diff
@@ -98,7 +98,7 @@
 
 :host([disabled]) {
 	opacity: var(--sapContent_DisabledOpacity);
-	pointer-events: unset;
+	pointer-events: none;
```

Comment:
Hi @Todor-ads, thank you for the fix, it was "none" for many years and it seems good, but recently has been changed with this change https://github.com/SAP/ui5-webcomponents/pull/5936
I shared my doubt in the PR, I see it was approved by you and merged. Just in case, could you sync with @unazko (if you did not yet.) if this breaks his fix.

Reply:
Also, could you add a test that cover the use-case, so that we don't introduce the same issue in future, Thank you